src/ocm/components/clusterConfiguration/staticIp/commonValidationSchemas.tsx

```diff
@@ -50,6 +60,65 @@ export const getIpAddressValidationSchema = (protocolVersion: 'ipv4' | 'ipv6') =
   );
 };
 
+export const compareIPV6Addresses = (address1: Address6, address2: Address6) => {
+  if (JSON.stringify(address1.toByteArray()) === JSON.stringify(address2.toByteArray())) {
+    return true;
+  }
+  return false;
+};
+
+export const isNotLocalHostIPAddress = (protocolVersion: 'ipv4' | 'ipv6') => {
```

Comment:
you can avoid the code duplication of isNotLocalHostIPAddress and isNotCatchAllIpAddress by creating a function getIsNotValidIpAddress that gets the invalid addresses, which is the dictionary LOCAL_HOST_IP or CATCH_ALL_IP. But this is just a code improvement you can leave this as is, it's good enough

Reply:
I wanted to them separately because I think they can be useful for other validations

Reply:
But it's good enough as is so I approved